code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -607,7 +607,7 @@
 		var/mob/living/carbon/C = M
 		var/holycheck = ishumanbasic(C)
 		if(reac_volume < 5 || !(holycheck || islizard(C) || (ismoth(C) && C.dna.features["moth_wings"] != "Burnt Off"))) // implying xenohumans are holy //as with all things,
-			if(method == INGEST && show_message)
+			if((method & INGEST) && show_message)
```

Comment:
change the var to methods to make sense that its a bitflag.

Reply:
and do this to every proc with the var too

Reply:
Done.